coreTable/CoreTableUtils.test.ts

```diff
@@ -258,3 +259,48 @@ describe(sortColumnStore, () => {
         expect(result["pops"]).toEqual([21, 99, 123])
     })
 })
+
+describe(concatColumnStores, () => {
+    it("concats stores with matching columns", () => {
+        expect(
+            concatColumnStores([
+                {
+                    a: [1, 2],
+                    b: [5, 6],
+                },
+                {
+                    a: [3, 4],
+                    b: [7, 8],
+                },
+            ])
+        ).toEqual({
+            a: [1, 2, 3, 4],
+            b: [5, 6, 7, 8],
+        })
+    })
+
+    it("concats column stores with missing columns", () => {
+        expect(
+            concatColumnStores([
+                {
+                    a: [1, 2],
+                    b: [6, 7],
+                },
+                {
+                    a: [3, 4],
+                    c: [0, 0],
```

Comment:
So we don't append columns, right? `c` is drop. I think this is reasonable, but maybe we should add a comment to the `concatColumnStores` method or even rename it to indicate that only the first columnStore's columns will be kept?